clients/src/test/java/org/apache/kafka/common/protocol/ApiKeysTest.java

```diff
@@ -49,8 +49,7 @@ public void schemaVersionOutOfRange() {
      * <ul>
      *   <li> Cluster actions used only for inter-broker are throttled only if unauthorized
      *   <li> SASL_HANDSHAKE and SASL_AUTHENTICATE are not throttled when used for authentication
-     *        when a connection is established. At any other time, this request returns an error
-     *        response that may be throttled.
+     *        when a connection is established or for re-authentication thereafter.
```

Comment:
The comment is now incomplete. Don't we still throttle in some cases?

Reply:
👍 I adjusted the Javadoc.  Let me know if it is now more accurate.